client/assets/stylesheets/_wp-components-overrides.scss

```
@@ -35,8 +39,9 @@
 	&:disabled,
 	&.disabled {
 		color: var(--color-neutral-20);
-		background-color: var(--color-surface);
+		background-color: var(--gray-gray-5, #dcdcde);
```

```suggestion
		background-color: var(--color-neutral-5);
```
It's the [same thing](https://github.com/Automattic/wp-calypso/blob/trunk/packages/calypso-color-schemes/src/shared/color-schemes/_default.scss#L73), just matching the border-color below it.